frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1341,13 +1346,16 @@ function BookMapWidget:showMenu()
     }
     UIManager:show(button_dialog)
 end
+
+function BookMapWidget: onShowBookMapMenu()
```

Comment:
I mean you can rename just
https://github.com/koreader/koreader/blob/c7d8781b77c12a797f999eaf1f7d43de44e2c482/frontend/ui/widget/bookmapwidget.lua#L1188
and do not introduce the additional method.

Reply:
done